config/locales/en.yml

```diff
@@ -479,44 +479,65 @@ en:
         one: "hour"
         other: "hours"
 
-  default_activity_management: "Management"
-  default_activity_design: "Design"
   default_activity_development: "Development"
+  default_activity_management: "Management"
+  default_activity_other: "Other"
+  default_activity_specification: "Specification"
+  default_activity_support: "Support"
   default_activity_testing: "Testing"
-  default_activity_Support: "Support"
-  default_activity_Other: "Other"
-  default_status_new: "new"
-  default_status_specified: "specified"
-  default_status_confirmed: "confirmed"
-  default_status_to_be_scheduled: "to be scheduled"
-  default_status_scheduled: "scheduled"
-  default_status_in_progress: "in progress"
-  default_status_tested: "tested"
-  default_status_on_hold: "on hold"
-  default_status_rejected: "rejected"
-  default_status_closed: "closed"
+  default_color_black: "Black"
+  default_color_blue: "Blue"
+  default_color_blue_dark: "Blue-dark"
+  default_color_blue_light: "Blue-light"
+  default_color_green_dark: "Green-dark"
+  default_color_green_light: "Green-light"
+  default_color_grey-dark: "Grey-dark"
+  default_color_grey-light: "Grey-light"
+  default_color_grey-medium: "Grey-medium"
+  default_color_magenta: "Magenta"
+  default_color_orange: "Orange"
+  default_color_red: "Red"
+  default_color_white: "White"
+  default_color_yellow: "Yellow"
+  default_status_closed: "Closed"
+  default_status_confirmed: "Confirmed"
+  default_status_developed: "Developed"
+  default_status_in_development: "In development"
+  default_status_in_progress: "In progress"
+  default_status_in_specification: "In specification"
+  default_status_in_testing: "In testing"
+  default_status_new: "New"
+  default_status_on_hold: "On hold"
+  default_status_rejected: "Rejected"
+  default_status_scheduled: "Scheduled"
+  default_status_specified: "Specified"
+  default_status_tested: "Tested"
+  default_status_test_failed: "Test failed"
+  default_status_to_be_scheduled: "To be scheduled"
```

Comment:
Even better question:

Should those defaults be translated at all?

1. Under which locale is the initialization run?
2. How do we make sure that this happens for the right locale?